addon/components/flash-message.js

```diff
@@ -107,5 +107,11 @@ export default Component.extend({
     if (flash) {
       flash.destroyMessage();
     }
+  },
+
+  actions: {
+    close() {
+      this._destroyFlashMessage();
```

Comment:
@sbatson5 keep in mind that `warn` is stripped from Production builds: https://emberjs.com/api/#method_warn